npm_and_yarn/spec/spec_helper.rb

```diff
@@ -9,3 +9,54 @@ def require_common_spec(path)
 end
 
 require "#{common_dir}/spec/spec_helper.rb"
+
+require_relative "./support/project_fixtures"
+
+require "parser/current"
+
+def walk(node, &block)
+  yield node
+
+  node.children.each do |child|
+    next unless child.is_a?(::Parser::AST::Node)
+
+    walk(child, &block)
+  end
+end
+
+RSpec.configure do |c|
+  c.after do
```

Comment:
Doing this after each test is probably a little slower than it needs to be, we can run this in an `after(:all)` block instead, since we store the results per test. We can get get rid of storing the `current_group` also, since it's no longer used.